dist/BingAdsEventForwarder.common.js

```diff
@@ -49,6 +59,8 @@ function isObject(val) {
         self.name = name;
 
         function initForwarder(settings, service, testMode) {
+            console.warn('BING BANG BOOM');
```

Comment:
remove this from this and the next build file if you do end up keeping this file in here